plutus-book/doc/09-extended.adoc

```diff
@@ -177,114 +225,64 @@ together out of transaction and ledger data,
-type:
-
-----
-data TxInType =
-      ConsumeScriptAddress !ValidatorScript !RedeemerScript
-    -- ^ A transaction input that consumes a script address with the given validator and redeemer pair.
-    | ConsumePublicKeyAddress !PubKey
-    -- ^ A transaction input that consumes a public key address.
-----
+The address of the funds locked by the contract is always the _validator script
+hash_. In this case, we could have included the hashed secret word directly
+in the validator. Making it part of the data script (state) of the contract
+makes the same validator usable for multiple word games (this is not the case
+for most other more complex contracts). The use of the
+redeemer, however, is inevitable here. It has to come from a participant,
+and represents the act of guessing the secret word.
 
-To recap the paying and spending process in a different way,
+We just saw the the extension to the UTxO and transaction data necessary
+to accommodate Plutus, and the different types of scripts used.
+Now we would like to recap the process of paying and spending to and from
+a script to solidify the reader's understand of the process:
 
 . Paying to a script address
 * Transaction contains an output `o` where the address in it is the hash of the
 validator script
 - this hash is computed off-chain by the wallet
 * `o` also contains the corresponding data script
-* `o` becomes a value in the UTxO map once the carrying transaction is processed
-* No validation occurs
-- Neither the full validator nor the redeemer are part of this payment process
+* `o` becomes a value in the ledger UTXO once the carrying transaction is processed
+* No validation occurs for paying to a script
+- The redeemer is not part of this payment process
 
 . Spending from a script address
 * Transaction contains an input `i` which
-- references the output in the UTxO it is spending
-- contains the full validator and redeemer scripts
-* The UTxO entry which `i` references is a script address, and thus has a
+- references the output in the UTXO it is spending
+- carries the full validator and redeemer scripts
+* The UTXO entry which `i` references is a script address, and thus has a
 data script stored in the output
-- the validator script `i` provides must hash to this script address
+- the validator script `i` provides hashes to this script address
 * Validation is done at this point for the spending of the specific output
 `i` references
 - we have the validator, redeemer and data script all available
-- if it succeeds, the transaction is allowed to spend the UTxO output `i` references
+- if it succeeds, the transaction is allowed to spend the UTXO output `i` references
 - validation is performed on-chain
 
-If the `runScript` computation returns true for a specific output being spent,
-the validation process proceeds.
+If the validation computation returns true for a specific script-address UTXO
+entry, the validation process proceeds.
```

Comment:
Validation doesn't apply to UTXO entries. Perhaps you mean for the particular pair of an input and an output?